packages/trader/src/AppV2/Components/DatePicker/date-picker.tsx

```diff
@@ -34,7 +50,7 @@ const DateRangePicker = ({ handleDateChange, isOpen, onClose, setCustomTimeRange
 
     return (
         <ActionSheet.Root isOpen={isOpen} onClose={onClose} position='left'>
-            <ActionSheet.Portal>
+            <ActionSheet.Portal shouldCloseOnDrag>
```

Comment:
Additional prop which I added to Quill according to our behaviour requirements. It'll close (not change height as was by default) the Action sheet